modules/otel-collector-components.adoc

```diff
@@ -199,6 +199,138 @@ The OpenCensus receiver provides backwards compatibility with the OpenCensus pro
 Wildcards with `+*+` are accepted under the `cors_allowed_origins`.
 To match any origin, enter only `+*+`.
 
+[id="k8scluster-receiver_{context}"]
+=== Kubernetes Cluster Receiver
+
+The Kubernetes Cluster Receiver gathers cluster metrics and entity events from the Kubernetes API server. It uses the Kubernetes API to receive information about updates. Authentication for this receiver is only supported through service accounts.
+
+.OpenTelemetry Collector custom resource with the enabled Kubernetes Cluster Receiver
+[source,yaml]
+----
+  receivers:
+    k8s_cluster:
+      distribution: openshift
+      collection_interval: 10s
+  exporters:
+    debug:
+  service:
+    pipelines:
+      metrics:
+        receivers: [k8s_cluster]
+        exporters: [debug]
+      logs/entity_events:
+        receivers: [k8s_cluster]
+        exporters: [debug]
+----
+
+This receiver requires a set up ServiceAccount, RBAC rules for the ClusterRole, and the ClusterRoleBinding that binds the RBAC with the ServiceAccount.
+
+.The ServiceAccount
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: otelcontribcol
+  name: otelcontribcol
+----
+
+.The RBAC rules for the ClusterRole
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Consider also the [collapsible option](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references) for this longest block, or even just all 3 for consistency.

Reply:
Thank you for the suggestion.
I like using the collapsible option where possible. However, I'll hold off adding complexity to the docs until migration to the new toolchain.